### ISSUE TITLE
Use same validation set each epoch

### ISSUE THREAD

**Issue by lewfish** [azavea/keras-semantic-segmentation#6]
Currently the validation set is shuffled and randomly augmented, which adds to the variance of the validation loss each epoch. It makes more sense to use the same validation set for each epoch so that epochs are more directly comparable.
